### PR TITLE
Updated Survey with multiple choice single select

### PR DESCRIPTION
**Pull request by Saurabh-Thakre**
Creating Survey with multiple choice single select using tower-cli

tower-cli job_template survey --name="Demo Job Template" --survey-spec='[{"question_name":"Test","question_description":"","required":true,"type":"multiselect","variable":"my_var","min":null,"max":null,"default":"choice1","choices":"choice1\nchoice2\nchoice3","new_question":true}]'

### CODE REVIEW

File: docs/source/cli_ref/usage/SURVEYS.rst [hunk 1]
```diff
@@ -94,3 +94,8 @@ Workflows
 Workflows can also have surveys and follow the same pattern. Example:
 
 ``tower-cli workflow survey --name="workflow with survey"``
+----------
+
+Configuring a survey with multiple choice single select :
+
+tower-cli job_template survey --name="Demo Job Template" --survey-spec='[{"question_name":"Test","question_description":"","required":true,"type":"multiselect","variable":"my_var","min":null,"max":null,"default":"choice1","choices":"choice1\nchoice2\nchoice3","new_question":true}]'
```
[the remark this answers is not in the document]
Comment (Saurabh-Thakre) [reply docs/source/cli_ref/usage/SURVEYS.rst]: @AlanCoding would you please look out at it

[the remark this answers is not in the document]
Comment (AlanCoding) [reply docs/source/cli_ref/usage/SURVEYS.rst]: oh, `modify` versus `survey` command. Yeah, that sounds right.

[the remark this answers is not in the document]
Comment (Saurabh-Thakre) [reply docs/source/cli_ref/usage/SURVEYS.rst]: Yeah !

[the remark this answers is not in the document]
Comment (Saurabh-Thakre) [reply docs/source/cli_ref/usage/SURVEYS.rst]: we should merge it to all the branches

**Pull request merged**